xonsh/environ.py

```diff
@@ -926,6 +928,15 @@ def _current_job():
     return ''
 
 
+def env_name(pre_chars='(', post_chars=') '):
+    """Extract the current environment name from $VIRTUAL_ENV."""
+    env_path = builtins.__xonsh_env__.get('VIRTUAL_ENV', '')
```

Comment:
This variable should not be set by default since there's no virtual environment activated by default. This is why I didn't put it in `DEFAULT_VALUES`.